steel/build.rs

```diff
@@ -0,0 +1,52 @@
+use sha2::{Digest, Sha256};
+use std::{env, fs::File, io::Write, path::Path};
+
+const TARGET_FILE: &str = "context.rs";
+const MAX_MERKLE_TREE_DEPTH: usize = 64;
+const BYTES_PER_CHUNK: usize = 32;
+
+fn hash_nodes(hasher: &mut Sha256, a: &[u8], b: &[u8], out: &mut [u8]) {
+    hasher.update(a);
+    hasher.update(b);
+    out.copy_from_slice(&hasher.finalize_reset());
+}
+
+fn compute_zero_hashes() -> [u8; MAX_MERKLE_TREE_DEPTH * BYTES_PER_CHUNK] {
+    let mut hasher = Sha256::new();
+    let mut buffer = [0u8; MAX_MERKLE_TREE_DEPTH * BYTES_PER_CHUNK];
+    for i in 0..MAX_MERKLE_TREE_DEPTH - 1 {
+        let focus_range = i * BYTES_PER_CHUNK..(i + 2) * BYTES_PER_CHUNK;
+        let focus = &mut buffer[focus_range];
+        let (source, target) = focus.split_at_mut(BYTES_PER_CHUNK);
+        hash_nodes(&mut hasher, source, source, target);
+    }
+    buffer
+}
+
```

Comment:
Would this comment be accurate?
```suggestion
NOTE: This build script is required by the vendored dependency ssz-rs.
```